src/server/auth/server/api_server.go

```diff
@@ -870,6 +870,31 @@ func (a *apiServer) GetRoleBinding(ctx context.Context, req *auth.GetRoleBinding
 	return response, nil
 }
 
+// GetRobotToken implements the protobuf auth.GetRobotToken RPC
+func (a *apiServer) GetRobotToken(ctx context.Context, req *auth.GetRobotTokenRequest) (resp *auth.GetRobotTokenResponse, retErr error) {
+	a.LogReq(req)
+	defer func(start time.Time) { a.LogResp(req, resp, retErr, time.Since(start)) }(time.Now())
+
+	subject := auth.RobotPrefix + req.Robot
```

Comment:
Would it make sense to canonicalize the subject? (i.e. in case a user accidentally sends `robot:foo` and then the token is for `robot:robot:foo` or something?)